app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,29 @@
+class PostsController < ApplicationController
+  require "will_paginate/array"
+  PAGE_SIZE = 4
+
+  def index
+    page = (params[:page] || '1').to_i
+
+    client = Tumblr::Client.new({
```

Comment:
I agree, Medium looks way nicer, but Jekyll requires git to update, right? We have volunteers who write blog posts who have zero terminal experience.